collector/tcpstat_linux_test.go

```diff
@@ -60,13 +60,6 @@ func Test_parseTCPStatsError(t *testing.T) {
 
 func TestTCPStat(t *testing.T) {
 
-	noFile, _ := os.Open("follow the white rabbit")
```

Comment:
This test should probably not be removed. We want to make sure we catch the error.

Reply:
Because these lines are replaced
```	if err != nil {
		return nil, err
	}
```

So that test is no longer valid

Reply:
The test is not obsolete, you need to check the scanner for errors.

Reply:
OK, i forgot to check scanner.Err().